### PR TITLE
Add visual comparison table component

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Add visual `ComparisonTable` to display in purchase form to show the difference between tiers.

## Justify why they are needed

Just stuck with a basic HTML table. It's only possible to make all columns the same width so that's why I added `min-width` property to the data cells to mimic the design. I think it works well enough.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

[GRW-1996]: https://hedvig.atlassian.net/browse/GRW-1996?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1216** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1216" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1216?utm_source=stack-comment).

File: apps/store/src/components/ProductPage/PurchaseForm/ComparisonTable/ComparisonTable.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    },
+  },
+}))
+
+type HeaderProps = { children?: React.ReactNode; className?: string; active?: boolean }
+
+export const Header = ({ children, active, ...props }: HeaderProps) => {
+  if (!children) return <TableHeader {...props} />
+
+  if (active) {
```
Comment (alebedev) [inline apps/store/src/components/ProductPage/PurchaseForm/ComparisonTable/ComparisonTable.tsx:33]: Nit: the only different thing about this subtree is header component.  You can make it conditional
```
const Header = active ? ActiveTableHeader : TableHeader
return (
  <Header {...props}>...</Header>
)
```

Or if active is intended to be props-driven, make TableHeader take it as a prop

**Review by alebedev — APPROVED** [review]
Nice!  One question iniside

File: apps/store/src/components/ProductPage/PurchaseForm/ComparisonTable/ComparisonTable.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import { ComponentMeta } from '@storybook/react'
+import * as ComparisonTable from './ComparisonTable'
+
+export default {
+  title: 'Product Page / Cancellation Form',
```
Comment (guilhermespopolin) [inline apps/store/src/components/ProductPage/PurchaseForm/ComparisonTable/ComparisonTable.stories.tsx:5]: Shouldn't this be something like "ComparisonTable"? 🤔

**Review by guilhermespopolin — APPROVED** [review]

**Pull request merged**